.travis.yml

```diff
@@ -1,29 +1,19 @@
+dist: bionic
 language: python
-dist: bionic   # required for Python >= 3.7
 cache: pip
-
-# Environment changes have to be manually synced with 'tox.ini'.
-# See: https://github.com/travis-ci/travis-ci/issues/3024
-
-python:
-  - "3.6"
-  - "3.7"
-
 matrix:
   include:
-    - python: 3.5
+    - python: "3.5"
       dist: xenial  # Bionic has no Python 3.5
-
+    - python: "3.6"
+    - python: "3.7"
     # Disabled, see https://github.com/sybrenstuvel/python-rsa/issues/131
     #- python: pypy3.5
     #  dist: xenial  # Bionic has no Python 3.5
-
+before_install: pip install --upgrade pip
 install:
   - pip install poetry
   - poetry install -v
-
-script:
-  - poetry run py.test tests/
-
-after_success:
-  - poetry run coveralls
+before_script: flake8 .
+script: poetry run py.test tests/
```

Comment:
The `script` and `after_success` sections haven't changed, you just moved it to one line. Please only include actual changes in the commit, and leave cleanups for another PR. This only makes the diff harder to read.

Reply:
It is more important that the file is concise and easier to read than the diff.  The diff will be read once, but the file hopefully many times.

Reply:
I disagree with that. Cleanups should go in a separate commit.